packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-code-build-multiple-inputs-outputs.expected.json

```diff
@@ -139,12 +139,14 @@
             {
               "Action": "sts:AssumeRole",
               "Effect": "Allow",
-              "Resource": {
-                "Fn::GetAtt": [
-                  "PipelineRoleD68726F7",
-                  "Arn"
-                ]
-              }
+              "Resource": [
```

Comment:
I'm a little surprised by this change. What about the implementation is overriding the `_norm` behavior of `PolicyStatement.toStatementJson`?

https://github.com/aws/aws-cdk/blob/2023004cc941a0e7a908bf3c90ad9887c6679564/packages/@aws-cdk/aws-iam/lib/policy-statement.ts?plain=1#L326-L356

My understanding is that any time we have a single resource, we remove the array.

Reply:
Great catch! Yeah, this is surprising behavior.

Let me dig in a little bit more to see what happens here.